src/dev/src/app/tree-view/tree-view.demo.scss

```diff
@@ -24,3 +24,31 @@
   overflow-y: auto;
   overflow-x: hidden;
 }
+
+.christmas ::ng-deep {
```

Comment:
This looks like [Christmas creep](https://en.wikipedia.org/wiki/Christmas_creep) to me.